src/main/java/pl/sdacademy/booking/mapper/ItemMapper.java

```diff
@@ -0,0 +1,35 @@
+package pl.sdacademy.booking.mapper;
+
+import pl.sdacademy.booking.data.ItemAttributeEntity;
+import pl.sdacademy.booking.data.ItemEntity;
+import pl.sdacademy.booking.model.ItemDto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ItemMapper {
+    public static ItemDto mapToItemDto(ItemEntity entity) {
+        if (entity == null) {
+            return null;
+        }
+
+        Set<String> attributes = mapAttributes(entity.getAttributes());
+        String name = (entity.getName() != null && !entity.getName().isEmpty()) ? entity.getName() : null;
```

Comment:
Variable 'name' is not used anywhere in the code. It should be replaced below...